src/jit/lowerxarch.cpp

```diff
@@ -2492,15 +2492,32 @@ bool Lowering::IsContainableHWIntrinsicOp(GenTreeHWIntrinsic* containingNode, Ge
 
         case HW_Category_SimpleSIMD:
         {
-            // These intrinsics only expect 16 or 32-byte nodes for containment
-            assert((genTypeSize(node->TypeGet()) == 16) || (genTypeSize(node->TypeGet()) == 32));
-            assert(supportsSIMDScalarLoads == false);
+            switch (containingIntrinsicId)
+            {
+                case NI_SSE41_ConvertToVector128Int16:
+                case NI_SSE41_ConvertToVector128Int32:
+                case NI_SSE41_ConvertToVector128Int64:
+                case NI_AVX2_ConvertToVector256Int16:
+                case NI_AVX2_ConvertToVector256Int32:
+                case NI_AVX2_ConvertToVector256Int64:
+                {
+                    supportsGeneralLoads = (!node->OperIsHWIntrinsic());
```

Comment:
Aligned and unaligned loads can't be supported since they only occur from the `Load` intrinsics, which require us to do a full load for deterministic behavior.

The memory encoding of the instructions will only do a partial vector read however (this is generally more than a scalar, but less than a full register).

Reply:
It appears to me that the memory encoding does a full vector read (full meaning 128 bits for SSE or the `VEX.128` encoding, and 256 bits for the `VEX.256` encoding). What am I missing?

Reply:
Only `VPMOVZXBW` does and, while we could special case that particular overload, it probably isn't worth doing it now:
![image](https://user-images.githubusercontent.com/10487869/56918149-20c35900-6a72-11e9-9ff6-4acb539f8f42.png)

Reply:
For example: `VPMOVZXBQ` is reading 4 packed bytes, equating to 32-bytes of memory (which is neither a scalar nor a full vector).